tests/lang_stmts/casestmt/tlabel_considers_converter.nim

```diff
@@ -0,0 +1,30 @@
+discard """
+  description: '''
+    Ensure that converters are considered with each 'of'-branch label syntax.
+  '''
+"""
+
+converter f2i(x: float): int8 =
+  int8(x)
+
+proc test(x: int8): int =
+  case x
+  of 1.0:
+    1
+  of 2.2 .. 3.3: # with range syntax: 2 .. 3
```

Comment:
I know this is an artifact of allowing for converters, it's still weird. The new behaviour is better, of course.